docs/source/examples/checks/vision/distribution/train_test_label_drift.ipynb

```diff
@@ -2,18 +2,84 @@
  "cells": [
```

Comment:
<p>can we make it show something interesting? Think Yuri has split this already</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/808/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>